### DISCUSSION TITLE
NGINX not starting

### DISCUSSION THREAD

**Discussion by suncoastDave** [beyondcode/herd-community#237] [General]
When I first installed Herd, it worked great for a week or so. Then I noticed that NGINX and DNSMasq were not starting (no green light). FPM is green. Looking at the logs, I get a could not bind, address already in use error on port 80. What could be interfering with it? Do I need to remove all other PHP and NGINX implementations on my box (MAMP and Homebrew-installed php)? Any other ideas? I've removed and then readded Herd to no avail.

**Comment by Lirux** [comment]
In my case, Valet was still started.
